nucleus/payara-modules/healthcheck-stuck/src/main/java/fish/payara/nucleus/healthcheck/stuck/admin/StuckThreadsConfigurer.java

```diff
@@ -43,6 +43,7 @@
 import com.sun.enterprise.util.LocalStringManagerImpl;
 import fish.payara.nucleus.healthcheck.HealthCheckService;
 import fish.payara.nucleus.healthcheck.admin.HealthCheckServiceConfigurer;
+import fish.payara.nucleus.healthcheck.admin.SetHealthCheckServiceConfiguration;
```

Comment:
Import just to link?

Reply:
Yeah, the `StuckThreadsHealthCheck` is in another package (and module) so it ends up like this.